packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1302,6 +1302,18 @@ export const UnknownFilesystemError = {
 	hint: 'It can be caused by many things, eg. missing permissions or a file not existing we attempt to read. Check the error cause for more details.',
 } satisfies ErrorData;
 
+/**
+ * @docs
+ * @description
+ * Your `tsconfig.json` `extends` property is invalid
+ */
+export const TSConfigInvalidExtends = {
+	name: 'TSConfigInvalidExtends',
+	title: 'Your `tsconfig.json` `extends` property is invalid',
+	message: 'Your `tsconfig.json` `extends` property is invalid',
```

Comment:
I don't think title and message should be the same. Use Erika's error guide! https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/README.md

Reply:
Yes, message should be an extended version of the title. "What happened, why etc."

Reply:
```suggestion
	title: 'Your `tsconfig.json` `extends` property is invalid',
	message: '`experimental.tsconfig` requires specific `extends` values inside `tsconfig.json`. Check out what change you need to make in the terminal above.',
```